cmd/minikube/cmd/start.go

```diff
@@ -135,6 +135,10 @@ func platform() string {
 
 // runStart handles the executes the flow of "minikube start"
 func runStart(cmd *cobra.Command, args []string) {
+	if cmd.Flag(rootless).Changed {
+		// env is internally used by kic runner
+		os.Setenv(constants.MinikubeRootlessEnv, strconv.FormatBool(viper.GetBool(rootless)))
```

Comment:
don't think we need to set env I believe viper lib already sets the env from the flags
check the other usages of Env as flag
```
MinikubeForceSystemdEnv = "MINIKUBE_FORCE_SYSTEMD"
```

so I believe we could delete this code

Reply:
viper seems to map `$MINIKUBE_ROOTLESS` to `--rootless` automatically, but it does not do vice versa.

Update the comment lines to clarify that.

Reply:
makes sense, thank you for the clarification